lib/tasks/stack_check.rake

```diff
@@ -7,9 +7,8 @@ namespace :stacks do
     next unless db.table_exists?(:buildpack_lifecycle_data)
```

Comment:
**suggestion** I don't think we need to check for the `buildpack_lifecycle_data` table anymore since we no longer need to load it because we don't load the stacks model.